docs/vm/cpu-pinning.md

```diff
@@ -105,8 +99,8 @@ To use CPU pinning on an existing VM, you must restart the VM after enabling the
 
 ## VM Live Migration
 
-VMs with CPU pinning enabled can be migrated only if the CPU Manager is enabled on the target node. For more information, see [Live Migration](./live-migration.md).
+VMs with CPU pinning enabled can be migrated only if the CPU Manager is enabled on the target node.
```

Comment:
I removed the reference to `For more information, see [Live Migration](./live-migration.md).` to avoid misleading users into thinking we provide additional information about live migration with CPU manager enabled, which we currently do not.